src/App.js

```diff
@@ -1,16 +1,83 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import SelectColor from './components/SelectColor';
+
+// iterate through chatMessages and create a list of senders
+const senders = [];
+for (const message of chatMessages) {
+  if (!senders.includes(message.sender)) {
+    senders.push(message.sender);
+  }
+}
+
+// assign first person in senders to local and second to remote (assumes only 2 senders and that the first sender is the local one, wouldn't work for group text!)
```

Comment:
Also wouldn't work if the chat was empty because senders[someIndex] would return undefined, but we're working with hardcoded data. Would be good to use a more dynamic (not hardcoding 0 and 1) approach for setting local and remote for the people chatting.